src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedPrimaryConstructor.cs

```diff
@@ -28,9 +28,13 @@ internal sealed class SynthesizedPrimaryConstructor : SourceConstructorSymbolBas
 
             this.MakeFlags(
                 MethodKind.Constructor,
+                RefKind.None,
                 containingType.IsAbstract ? DeclarationModifiers.Protected : DeclarationModifiers.Public,
                 returnsVoid: true,
+                hasAnyBody: false,
```

Comment:
>false [](http://example.com/codeflow?start=28&length=5)

It feels to me that `true` is a better choice here. It is never an abstract or external method.

Reply:
done.